docs/data/capture/performance-metrics.md

```diff
@@ -37,6 +37,18 @@ The module provides a [sensor](/components/sensor/) that allows you to obtain re
 1. Go to your machine's **Config** page and click **Create component**.
 2. Then select the `viam:viam-sensor:telegrafsensor` model from the [`viam-telegraf-sensor` module](https://app.viam.com/module/viam/viam-telegraf-sensor).
 3. Click **Add module**, then enter a name for your sensor, for example `my-telegrafsensor`, and click **Create**.
+4. To enable or disable specific metrics, add them to the attributes configuration.
+   For example:
+
+   ```json
+   {
+     "disable_kernel": true
+   }
+   ```
+
+   For all the configuration options see [the module documentation](https://github.com/viamrobotics/viam-telegraf-sensor?tab=readme-ov-file#attributes).
```

Comment:
Opt nit: clarifying specific module docs, not "docs for modules" (like /registry/module is what I thought of first).

```suggestion
   See [the `telegrafsensor` module documentation](https://github.com/viamrobotics/viam-telegraf-sensor?tab=readme-ov-file#attributes) for the full list of attributes available.
```